server/README.md

```diff
@@ -1,8 +1,32 @@
-## How to see our database?
+## How run the backend?
+
+Go into our server folder.
+`cd server`  
+Install dependencies
+`yarn install`
+
+Configure `.env` file.
+
+1. Create a new file `.env` in the `server` folder at the root level.
```

Comment:
Hey Andy, if you'd like to, we can replace this step with something along the lines of:
"Run the command ```touch .env``` to create a new file `.env` in the `server` folder"